src/Microsoft.Azure.SignalR.AspNet/ClientConnections/ClientConnectionManager.cs

```diff
@@ -62,6 +64,13 @@ public bool TryGetServiceConnection(string key, out IServiceConnection serviceCo
             return _clientConnections.TryGetValue(key, out serviceConnection);
         }
 
+        public bool TryRemoveServiceConnection(string connectionId, out IServiceConnection connection)
+        {
+            return _clientConnections.TryRemove(connectionId, out connection);
+        }
+
+        public IReadOnlyDictionary<string, IServiceConnection> ClientConnections => _clientConnections;
```

Comment:
>ClientConnections [](start = 63, length = 17)

only use for tests?

Reply:
For now it is. Intent to keep to some extent consistent with the core version.